attest/attest.go

```diff
@@ -132,11 +161,18 @@ func Attest(attestation []byte, nonce []byte, rootCert *x509.Certificate) (*Atte
 		return nil, err
 	}
 
-	if rootCert != nil {
-		if err := verifyCertChain(cert, rootCert, doc.Cabundle); err != nil {
-			log.Errorf("Error verifying certificate chain: %v", err)
+	rootCert := v.rootCert
+	if rootCert == nil {
+		c, err := GetRootAWSCert()
```

Comment:
Not a blocker: Since this involves a network access, it's worth considering whether we could effectively cache this cert by also storing it in `v.rootCert`.

Reply:
Good idea, just to merge for right now. Will open an issue.